universal-application-tool-0.0.1/app/views/J2HtmlDemoView.java

```diff
@@ -35,7 +35,8 @@ public Content render(String greeting, Request request) {
             form(
                     makeCsrfTokenInputTag(request),
                     FieldWithLabel.input()
-                        .setId("firstName")
+                        .setId("first-name-input")
```

Comment:
suggestion (non-blocking): I tend to prefer classes to IDs in general. In this case specifically, I can imagine a scenario where we have two first name inputs on the same page (e.g., "you and your spouse's names).

Thoughts on using class instead of id throughout?

I realized after writing the above how annoying it might be given the way that J2Html overrides classes on each call to `withClass(es)` though 😕  So feel free to ignore if that hurdle seems too big.